tests/test_taking_a_vacation.py

```diff
@@ -105,20 +100,16 @@ def test_trip_cost_negative(input_city, input_days, input_spending_money):
     ("Los Angeles", 365, 66125)
 ])
 def test_trip_cost_only(input_city, input_days, expected):
-    try:
-        assert input_city in ("Charlotte", "Tampa", "Pittsburgh", "Los Angeles")
-        assert type(input_days) == int and input_days > 0,\
-            "{} was not int or positive integer > 0".format(input_days)
-        assert trip_cost_only(input_city, input_days) == expected
-    except AssertionError as e:
-        print(e)
+    assert trip_cost_only(input_city, input_days) == expected
 
 
 @pytest.mark.parametrize("input_days", [-1, "blah", False, None])
-@pytest.mark.parametrize("input_city", ["Sochi", "Tampa"])
+@pytest.mark.parametrize("input_city", ["Sochi"])
```

Comment:
no need to use parametrization if you have only 1 value.
Just use that value in the call to the function under test.